base/setup/usetup/usetup.c

```diff
@@ -3360,9 +3361,14 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
      */
     if ((RepairUpdateFlag || IsUnattendedSetup) && IsValidPath(InstallDir))
     {
-        BuildInstallPaths(InstallDir,
-                          DiskEntry,
-                          PartEntry);
+        Status = BuildInstallPaths(InstallDir,
+                                   DiskEntry,
+                                   PartEntry);
+
+        if (!NT_SUCCESS(Status))
+        {
+            DPRINT1("BuildInstallPaths() failed. Status code: 0x%lx", Status);
```

Comment:
Because installation cannot continue you should popup an error and quit the installer. You have an example how to display a popup error below, but then you need to `return QUIT_PAGE;` instead.

Reply:
Do you think `Failed to build the installation paths for the ReactOS installation directory!` would fit as an error message reflecting the failure? Are you OK on localizing such string or should I do this in a separate PR?

Reply:
The error message is fine. You can make the localization in a separate PR.